components/ReleaseStats/ReleaseStats.jsx

```diff
@@ -0,0 +1,163 @@
+import { useState } from "react"
+import { createClientComponentClient } from "@supabase/auth-helpers-nextjs"
+import {
+  Dialog,
+  DialogTrigger,
+  DialogContent,
+  DialogPortal,
+  DialogClose,
+} from "@/components/Dialog/Dialog"
+
+let today = new Date()
+
+let yesterday = new Date()
+yesterday.setDate(yesterday.getDate() - 1)
```

Comment:
This seems like it could create a bug. I think you'll want to pass `today` as an argument to each of the following variables to ensure that they match:

```js
let today = new Date()

let yesterday = new Date(today)
yesterday.setDate(yesterday.getDate() - 1)

// ...and so on
```

Reply:
Thats a good call.